include/mbgl/util/string.hpp

```diff
@@ -13,6 +13,19 @@ inline std::string toString(T t) {
     return std::to_string(t);
 }
 
+inline std::string toString(double num) {
+    if (num - (int)num > 0.0 || num + (int)num < 0.0) {
```

Comment:
I think this can be expressed as `if (std::fmod(num, 1))`. `std::fmod()` and `fmod()` are floating-point-friendly versions of the `%` operator.